source/gui/nvdaControls.py

```diff
@@ -68,22 +68,6 @@ def sendListItemFocusedEvent(self, index):
 		evt.Index = index
 		self.ProcessEvent(evt)
 
-class SelectOnFocusSpinCtrl(wx.SpinCtrl):

```

Comment:
Unless you expect this to be merged before 2021.1 this class should stay since some add-ons may rely on it (certainly one of mine does). Since we don't  know if selecting content of spin controls by default was a deliberate decision on the WX  Vidgets side it would be good to have a system test ensuring that this behaviour persists when we update wxPython. My preference would be for both removal of this class  and eventual automated test to happen in a separate pr.

Reply:
> Unless you expect this to be merged before 2021.1 this class should stay since some add-ons may rely on it (certainly one of mine does). Since we don't know if selecting content of spin controls by default was a deliberate decision on the WX Vidgets side it would be good to have a system test ensuring that this behaviour persists when we update wxPython. My preference would be for both removal of this class and eventual automated test to happen in a separate pr.

Good point. Thanks. When the remaining issue of this PR will be solved, I will:
* either add an additional note about deprecation in the change log if this PR can still be included in 2021.1
* or revert the commit that removed the auto-select component if this PR may not be merged in 2021.1 anymore